lib/service/confirmation_dialogs/custom_confirmation_dialog.dart

```diff
@@ -0,0 +1,194 @@
+import 'package:ax_dapp/service/controller/controller.dart';
```

Comment:
I think for this file name, transaction_confirmed_dialog.dart is better than now.

Reply:
done